### ISSUE TITLE
Does your app sell usage of phone numbers for SMS-receiving purposes?

### ISSUE THREAD

**Issue by Simon311** [moezbhatti/qksms#1845]
Over the past year or two, there's been a weird case of my phone number seemingly being used to make accounts that I didn't make, even so far as someone successfully registering my phone number with the Russian Post office.
My device runs latest Motorola firmware and only your app, installed from F-Droid, has access to SMS. 

What's up with that?

**Comment by GithubContributor832** [comment]
Read the privacy policy bud. Fdroid version has no trackers in the app. Are you using a privacy friendly contacts app? Many contacts apps make a copy of your contacts and use them unethically.

**Comment by Simon311** [comment]
Yeah this really isn't funny. It actually happened again, someone successfully registered an Aliexpress account with my number. Answer the question: are you doing this?

**Comment by Simon311** [comment]
It seems I am not the only one experiencing this issue. https://4pda.to/forum/index.php?showtopic=576815&view=findpost&p=105851339
![image](https://user-images.githubusercontent.com/1058851/151103666-6882e75f-a927-4d0d-a7fa-48ad2513a647.png)

**Comment by licaon-kter** [comment]
Ref: https://forum.f-droid.org/t/qksms-compromised/16741

I'll reiterate, just because they try doesn't mean they succeed....

**Comment by Zulgrib** [comment]
For sure Fdroid would not inject that, meaning one could find proofs in the source code.

**Comment by e-t-l** [comment]
Yeah, you're free to read through the source code. There's nothing there that could possibly record your phone number and send it to any server. Lot sof apps can read your device's phone number without needing SMS access, fyi. Also, if you're ever suspicious that an app on your phone is secretly sending info to a server, you can install a DNS-over-VPN like Blokada or Netguard to view and block the connections any app makes.

But your particular issue might be even simpler. Consider it this way: If you are in the US, you have a 10 digit phone number. If someone somewhere in the world wants to create a throwaway account that asks for a phone number, they can enter any combination of ten digits. They might enter _your_ phone number, even if they don't specifically know it's _yours_.
